project/CONTRIBUTING.md

```diff
@@ -87,9 +87,16 @@ Some rules have to be respected about the test:
   * `@codeCoverageIgnore`
   * `@codeCoverageIgnoreStart`
   * `@codeCoverageIgnoreEnd`
+  * `@expectedException`
+  * `@expectedExceptionMessages`
 * All test methods should be prefixed by `test`. Example: `public function testItReturnsNull()`.
+* All test method names must be in camel case format
 * As opposed, the `@test` annotation is prohibited.
 * Most of the time, the test class should have the same name as the targeted class, suffixed by `Test`.
+* `@expectedException*` annotations are prohibited in favor of:
+  * the `setExpectedException` method if phpunit 4 is still in use;
```

Comment:
Didn't see the previous line format. In this case:

```markdown
The `@expectedException*` annotations are prohibited. Use `TestCase::setExpectedException` instead.
```

No need to talk about `expectException*` right now IMHO, all project are currently tested with PHPUnit 4.x.

We could change it later.